mix.exs

```diff
@@ -60,7 +61,8 @@ defmodule Kwtool.MixProject do
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:nimble_phx_gen_template, "~> 2.2.0", only: :dev, runtime: false}
+      {:argon2_elixir, "~> 2.0"},
+      {:faker_elixir_octopus, "~> 1.0.0", only: [:dev, :test]}
```

Comment:
Maybe you could check out this lib, it's more popular - https://github.com/elixirs/faker. Both works for me though 😆

Reply:
yes, thanks for suggestion, i will update in another PR, in this PR i am focusing on the backend first :)